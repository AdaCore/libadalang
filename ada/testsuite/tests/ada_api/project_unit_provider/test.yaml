driver: ada-api
main: main.adb
input_sources: [
    p.gpr,
    src1/p1.ada_specification,
    src2/p2.ada_specification
]
expect_failure:
    - {if: [valgrind], then: 'Pending resolution of S912-003'}
