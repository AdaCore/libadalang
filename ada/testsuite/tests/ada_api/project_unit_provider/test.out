Loading unsupported_aggr.gpr...
   Invalid_Project exception: aggregate project has too many sub-projects
Loading unsupported_aggr.gpr...
   Targetting subproject p
   Success
Loading supported_simple_aggr.gpr...
   Success
Loading supported_chained_aggr.gpr...
   Success
Loading p.gpr...
<SubtypeIndication p2.ada_specification:4:8-4:22> resolves to:
  <TypeDecl ["Record_Type"] p1.ada_specification:2:4-2:36>
Done.
